installer/resources/notification/s3_upload.py

```diff
@@ -0,0 +1,44 @@
+from core.terraform.resources.aws.s3 import S3BucketObject
+from resources.s3.bucket import BucketStorage
+from core.terraform.utils import get_terraform_scripts_and_files_dir
+from core.config import Settings
+from resources.pacbot_app.build_ui_and_api import BuildUiAndApis
+import os
+
+FETCH_NOTIFICATION = "send-notification-service"
```

Comment:
@rnithinpaladin I would suggest that we come up with a standard naming convention for a particular domain:

for example, everything related to notifications can follow something like:

[action]-[destination]-notification-service, which would translate to:

send-notification-service (assuming this is a lambda between SNS and our email service), would become -> send-email-notification-service
template-formatter-service -> template-formatter-notification-service
invoke-notification-service -> invoke-notification-service
paladincloud-inapp-notification -> inapp-notification-service

Reply:
Okay, And jar file name should be the same.